pkg/nuctl/command/get.go

```diff
@@ -99,30 +101,49 @@ func newGetFunctionCommandeer(getCommandeer *getCommandeer) *getFunctionCommande
 				return nil
 			}
 
+			rendererFunc := commandeer.renderFunctionConfig
+			if commandeer.withStatus {
```

Comment:
Why not always?

Reply:
not the current default behavior, status may contain deploy logs and can be very long. will leave it as false for now.
will add a todo to default it